grape-entity.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = 'grape-entity'
 
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activesupport', '<= 4.2.6'
```

Comment:
I'll take this change as is for now, and we (you) should open an issue to support latest AS. 

Can you amend the changelog entry please and I'll merge?